pkg/monitors/cpu/cpu.go

```diff
@@ -188,7 +188,8 @@ func getUtilization(prev *totalUsed, current *totalUsed) (utilization float64, e
 			err = fmt.Errorf("percent %v < 0 total: %v used: %v", utilization, usedDiff, totalDiff)
 		}
 		if utilization > 100.0 {
-			err = fmt.Errorf("percent %v > 100 total: %v used: %v ", utilization, usedDiff, totalDiff)
+			utilization = 100
+			m.logger.Warningf("actual percent %v > 100, total: %v, used: %v, sent value: 100", utilization, usedDiff, totalDiff)
```

Comment:
Hmm I'm not sure if we want to log if this can happen just due to rounding errors and spams the logs. 🤔 idk

Reply:
Yeah, fair enough. I made a quick change and didn't realize this was a function instead of a method, so didn't have access to the logger anyway.